### ISSUE TITLE
no-named-as-default-member -TypeError: Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by SimenB** [benmosher/eslint-plugin-import#281]
no-named-as-default-member.js:85:42

Unsure what source makes it crash... Narrowed it down to:

```js
function someFunction() {
  switch (someVariable) {
    case 'something': {
      const { // The error is reported at the { on this line
        ...rest
      } = someGlobal;
    }
  }
}
```

**Comment by benmosher** [comment]
Agh, must be the destructuring spread. Thanks for the heads up. 

I think I want to build up cases like this and run against every rule to catch these kinds of gotchas. 

Thanks for the tip!
